content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,162 @@
+
+- HudsonIsLoading / HudsonIsRestarting
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_a.png[role="center"]
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_b.png[role="center"]
+
+- Login
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_2.png[role="center"]
+
+- Sign up
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_3.png[role="center"]
+
+
+== UX enhancement
+
+Form validation has changed to give inline feedback about data validation errors in the same form.
+
+- Login
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_4.png[role="center"]
+
+- Sign up
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_5.png[role="center"]
+
+The above image shows that the validation is now done on all input fields instead of before breaking on the
+first error found, which should lead to less retry cycles.
+
+Instead of forcing the user to repeat the password, the new UX introduces the possibility to display the password in
+clear text. Further a basic password strength meter indicates the user the password strength while she is writing it.
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_6.png[role="center"]
+
+== SimplePageDecorator
+
+NOTE: The re-/starting screens do not support the concept of decorators very well hence the decision to drop them.
+
+The SimplePageDecorator uses three different files to allow overriding the look and feel of the login and signup pages.
+
+- simple-head.jelly
+- simple-header.jelly
+- simple-footer.jelly
+
+All of the above SimplePageDecorator jelly files are supported in the *login* page:
+
+[source,html]
+----
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" >
+  <j:new var="h" className="hudson.Functions"/>
+  <html>
+    <head>
+      <!-- css styling, will fallback to default implementation -->
+      <st:include it="${h.simpleDecorator}" page="simple-head.jelly" optional="true"/>
+    </head>
+    <body>
+      <div class="simple-page" role="main">
+        <st:include it="${h.simpleDecorator}" page="simple-header.jelly" optional="true"/>
+      </div>
+      <div class="footer">
+        <st:include it="${h.simpleDecorator}" page="simple-footer.jelly" optional="true"/>
+      </div>
+    </body>
+</j:jelly>
+----
+
+The *sign-up* page only supports the simple-head.jelly:
+
+[source,html]
+----
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" >
+  <j:new var="h" className="hudson.Functions"/>
+  <html>
+    <head>
+      <!-- css styling, will fallback to default implementation -->
+      <st:include it="${h.simpleDecorator}" page="simple-head.jelly" optional="true"/>
+    </head>
+</j:jelly>
+----
+
+== SimplePageDecorator - custom implementations
+
+NOTE: Have a look at https://github.com/daniel-beck/login-theme-plugin[login-theme-plugin], which allows you to
```

Comment:
I would vote for hosting it, seems useful even in the current state

Reply:
I pinged you about that in JENKINS-52000 😃

Reply:
I am +1 ;)